test/integration/govuk_index/specialist_formats_test.rb

```diff
@@ -0,0 +1,90 @@
+require 'integration_test_helper'
+
+class SpecialistFormatTest < IntegrationTest
+  def setup
+    super
+
+    bunny_mock = BunnyMock.new
+    @channel = bunny_mock.start.channel
+
+    consumer = GovukMessageQueueConsumer::Consumer.new(
+      queue_name: "bigwig.test",
+      processor: GovukIndex::PublishingEventProcessor.new,
+      rabbitmq_connection: bunny_mock
+    )
+
+    @queue = @channel.queue("bigwig.test")
+    consumer.run
+  end
+
+  def test_specialist_publisher_finders_are_correctly_indexed
+    random_example = GovukSchemas::RandomExample
+                       .for_schema(notification_schema: "finder")
+                       .merge_and_validate(document_type: "finder")
+    GovukIndex::MigratedFormats.stubs(:indexable_formats).returns(["finder"])
+
+    @queue.publish(random_example.to_json, content_type: "application/json")
+
+    assert_document_is_in_rummager({ "link" => random_example["base_path"] }, index: "govuk_test", type: 'edition')
+  end
+
+  def test_specialist_documents_are_correctly_indexed
+    document_types = %w(
+      aaib_report
+      asylum_support_decision
+      business_finance_support_scheme
+      cma_case
+      countryside_stewardship_grant
+      dfid_research_output
+      drug_safety_update
+      employment_appeal_tribunal_decision
+      employment_tribunal_decision
+      international_development_fund
+      maib_report
+      medical_safety_alert
+      raib_report
+      service_standard_report
+      tax_tribunal_decision
+      utaac_decision
+      vehicle_recalls_and_faults_alert
+    )
+
+    # ideally we would run a test for document types, but this takes 3 seconds so I have limited
```

Comment:
assume this is meant to say "for all document types"?

Reply:
corrected